src/api/utils/classification.py

```diff
@@ -0,0 +1,181 @@
+import datetime
+import hashlib
+import os
+import uuid
+from enum import Enum
+from io import BytesIO
+from typing import Any, Dict, Optional
+
+import pytz
+from django.contrib.gis.geos import Point
+from django.core.files.base import ContentFile
+from django.db.models.fields.files import ImageFieldFile
+from exif import Image as ExifImage
+from PIL import ExifTags, Image as PILImage
+from PIL.ExifTags import TAGS
+
+from ..models import Image
+
+
+def create_unique_image_name(image: ImageFieldFile) -> str:
+    image_name = image.name
+    image_ext = os.path.splitext(image_name)[1]
+    current_utc_time = datetime.datetime.now(datetime.timezone.utc)
+    uid = str(uuid.uuid4()).replace("-", "")
+    name = f"{uid}{current_utc_time:%Y%m%d_%H%M%S}"
+
+    return f"{name}{image_ext}"
+
+
+def create_image_checksum(image: ImageFieldFile) -> str:
+    if image.closed:
+        image.open("rb")
+    file_hash = hashlib.sha256()
+
+    for chunk in image.chunks():
+        file_hash.update(chunk)
+
+    return file_hash.hexdigest()
+
+
+def create_thumbnail(image: ImageFieldFile) -> ContentFile:
+    img = PILImage.open(image)
+    size = (100, 100)
```

Comment:
Just checking that these settings (with respect to aspect ratio, resolution, etc.) are more or less equivalent to CoralNet's?